packages/components/app/styles/components/side-nav.scss

```diff
@@ -0,0 +1,238 @@
+//
+// SIDE-NAV
+//
+
+@use "../mixins/focus-ring" as *;
+
+// WIP, naming, etc. will be refined... TODO: Add JSON for color tokens, etc.
+:root {
+  /* Sizes: */
+
+  // header, body, footer "wrappers":
+  --token-side-nav-wrapper-padding-horizontal: 16px;
+  --token-side-nav-wrapper-padding-vertical: 16px;
+
+  // Body sections
+  --token-side-nav-body-section-margin-vertical: 24px;
+
+  // Items/Links in body:
+  --token-side-nav-section-item-padding-horizontal: 8px;
+  --token-side-nav-section-item-padding-vertical: 4px;
+
+  /* Colors: */
+
+  /* SideNav container: */
+  --token-side-nav-foreground-primary: #dedfe3;
+  --token-side-nav-foreground-primary-strong: #fff;
+  --token-side-nav-foreground-faint: #8c909c;
+  --token-side-nav-surface: #0c0c0e;
+
+  // Interactive elements:
+  // base:
+  --token-side-nav-interactive-surface-hover: #3b3d45;
+  --token-side-nav-interactive-surface-active: #656a76;
+
+  // selected / current:
+  --token-side-nav-interactive-selected-surface-default: #656a76;
+  --token-side-nav-interactive-selected-surface-hover: #3b3d45;
+  --token-side-nav-interactive-selected-surface-active: #656a76;
+}
+
+
+// * SideNav Parent (wrapper) component
+// -------------------------------------------------------------------
+
+.hds-side-nav__wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  color: var(--token-side-nav-foreground-primary); // Default color
+  background: var(--token-side-nav-surface);
+}
+
+.hds-side-nav__wrapper-header {
+  padding: var(--token-side-nav-wrapper-padding-vertical) var(--token-side-nav-wrapper-padding-horizontal);
+}
+
+.hds-side-nav__wrapper-body {
+  flex: 1;
+  padding: var(--token-side-nav-wrapper-padding-vertical) var(--token-side-nav-wrapper-padding-horizontal);
+  overflow-y: auto;
+}
+
+.hds-side-nav__wrapper-footer {
+  padding: var(--token-side-nav-wrapper-padding-vertical) var(--token-side-nav-wrapper-padding-horizontal);
+}
+
+// * Header child elements
+// ---------------------------------------
+
+.hds-side-nav-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.hds-side-nav-header__logo {
+  display: block;
+  align-items: center;
+  justify-content: center;
+  width: 48px;
+  height: 48px;
+}
+
+.hds-side-nav-header__actions {
+  display: flex;
+  gap: 16px;
```

Comment:
@cristiano Should I replace this `gap` style with margins?

Reply:
yes, for now.

Reply:
I replaced it with a in-between margin style. Let me know if you don't like the general selectors I used though.